scripts/src/cpu.lua

```diff
@@ -1618,8 +1618,8 @@ if CPUS["M6502"] then
 		MAME_DIR .. "src/devices/cpu/m6502/m7501.h",
 		MAME_DIR .. "src/devices/cpu/m6502/m8502.cpp",
 		MAME_DIR .. "src/devices/cpu/m6502/m8502.h",
-		MAME_DIR .. "src/devices/cpu/m6502/n2a03.cpp",
-		MAME_DIR .. "src/devices/cpu/m6502/n2a03.h",
+		MAME_DIR .. "src/devices/cpu/m6502/rp2a03.cpp",
+		MAME_DIR .. "src/devices/cpu/m6502/rp2a03.h",
 		MAME_DIR .. "src/devices/cpu/m6502/r65c02.cpp",
```

Comment:
These are now out of alphabetical order.  Same for the rest of the changes in this file.

Reply:
It wasn't exactly ordered to begin with. It's a bit more sorted now.